packages/cli-dashboard/src/components/utils/generateReports/generateAllCookiesCSV.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { sanitizeCsvRecord } from '@ps-analysis-tool/common';
+/**
+ * Internal dependencies
+ */
+import type { CompleteJson, CookieJsonDataType } from '../../../types';
+
+export const cookieDataHeader = [
```

Comment:
Capitalized Snack Case.

Reply:
Second this idea,  but do we need to export this? I don't see any reference to this variable outside this file.